longruntests/longruntests_suite_test.go

```diff
@@ -0,0 +1,404 @@
+package longruntests_test
```

Comment:
Is this file a wholesale copy of https://github.com/deis/workflow-e2e/blob/master/tests/tests_suite_test.go ?  if so, can we just use that?  

If no, can we just add any `longrun`-specific functions into the aforementioned file?  If the majority of the code is identical, definitely want to see it in one place only...